examples/mqtt-client-basic-authentication/mqtt_client_basic_authentication.md

```diff
@@ -0,0 +1,21 @@
+# MQTT client - Basic authentication
+
+The `mqtt:Client` connects to an MQTT server via basic authentication and then, sends messages to the server. Basic authentication can be enabled by configuring the `username`, `password` in the `connectionConfig` of the `mqtt:ClientConfiguration`. Use this to connect to an MQTT server secured with basic authentication.
+
+::: code mqtt_client_basic_authentication.bal :::
+
+## Prerequisites
```

Comment:
```suggestion
## Prerequisites
```
Shall we have these steps as numbered bullet points?

Reply:
This is the format other stdlib bbes are in.